fix_proto_conf_desc.py

```diff
@@ -0,0 +1,100 @@
+import ida_segment
+import idautils
+import idc
+import ida_bytes
+
+def parse_pcd(pcd_addr):
+
+	#create comments for the PCD struct
+	ida_bytes.create_dword(pcd_addr, 4)
```

Comment:
A much better way would be to define a C struct, and simply setting the global to this type instead of going by each member manually. It will also look much better within IDA